.github/workflows/main.yml

```diff
@@ -29,6 +29,11 @@ jobs:
       uses: ./.github/actions/get-events-action
       with:
         crawler-name: 'ItalyCrawler'
+    - id: DEPodcast
+      name: Get DEPodcast events action step
+      uses: ./.github/actions/get-events-action
+      with:
+        crawler-name: 'DataEngineeringPodcastCrawler'
```

Comment:
@kunaljubce The DEPodcast has to go through to this crawler name, filenames, class etc.

Reply:
Hi @vinayak-mehta I have done the changes. Kindly check.